addons/mail/static/src/discuss/call/common/call.js

```diff
@@ -234,7 +250,8 @@ export class Call extends Component {
             return;
         }
         const { width, height } = this.grid.el.getBoundingClientRect();
-        const aspectRatio = this.minimized ? 1 : 16 / 9;
+        const aspectRatio =
+            (this.minimized && !this.env.inChatWindow) || this.state.collapsed ? 1 : 16 / 9;
```

Comment:
the new `!this.env.inChatWindow` breaks the style in chat window, chat window participants cards are circle(/squares) so they should stay as 1 aspect ratios.